### PR TITLE
[expo-updates][e2e] Fix e2e test

### PR DESCRIPTION
**Pull request by wschurman**
# Why

Borked by recent new dependency of expo-updates.

Example borken run: https://github.com/expo/expo/runs/5710956061?check_suite_focus=true

# How

Add local dependency.

# Test Plan

Wait for CI to run e2e test, ensure it works.

### CODE REVIEW

File: .github/workflows/updates-e2e.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ jobs:
         run: expo-cli init updates-e2e --yes
       - name: Add local expo-updates and dependencies
         working-directory: ../updates-e2e
-        run: yarn add file:../expo/packages/expo-updates file:../expo/packages/expo file:../expo/packages/expo-modules-core file:../expo/packages/expo-json-utils file:../expo/packages/expo-manifests file:../expo/packages/expo-structured-headers file:../expo/packages/expo-updates-interface
+        run: yarn add file:../expo/packages/expo-updates file:../expo/packages/expo file:../expo/packages/expo-modules-core file:../expo/packages/expo-json-utils file:../expo/packages/expo-manifests file:../expo/packages/expo-structured-headers file:../expo/packages/expo-updates-interface file:../expo/packages/expo-eas-client-id
```
Comment (ide) [inline .github/workflows/updates-e2e.yml:66]: Looks like this list is intended to be alphabetically ordered, otherwise LGTM

**Review by ide — APPROVED** [review]

File: .github/workflows/updates-e2e.yml (hunk 1 above)
Comment (wschurman) [reply .github/workflows/updates-e2e.yml:66]: The first package listed is `expo-updates` so probably not alphabetically ordered?

Comment (ide) [reply .github/workflows/updates-e2e.yml:66]: In any case, let's let the current CI run complete.

Comment (wschurman) [inline .github/workflows/updates-e2e.yml:66]:
```suggestion
        run: yarn add file:../expo/packages/expo-updates file:../expo/packages/expo file:../expo/packages/expo-eas-client-id file:../expo/packages/expo-modules-core file:../expo/packages/expo-json-utils file:../expo/packages/expo-manifests file:../expo/packages/expo-structured-headers file:../expo/packages/expo-updates-interface
```

**Pull request merged**